Bermuda/Bermuda/AggressiveBehaviour.cpp

```diff
@@ -0,0 +1,45 @@
+#include "AggressiveBehaviour.h"
+#include "PlayState.h"
+
+AggressiveBehaviour::AggressiveBehaviour(StateMachine<Entity>* m_pStateMachine) :
+	BaseBehaviour(m_pStateMachine)
+{
+
+}
+
+void AggressiveBehaviour::executeAction()
+{
+	
+	dynamic_cast<AttackingNPC*>( this->npc )->attack();
```

Comment:
1. Dat is een van de dingen die nog even aan gepast moet worden
2. En dat is een kwestie van smaak